search-engine/lib/card_printing.rb

```diff
@@ -27,15 +27,18 @@ def initialize(card, set, data)
     @rarity_code = %W[basic common uncommon rare mythic special].index(rarity) or raise "Unknown rarity #{rarity}"
     @frame = begin
       eight_edition_release_date = Date.new(2003,7,28)
+      m15_release_date = Date.new(2014,7,18)
       if @release_date < eight_edition_release_date
         "old"
       elsif @set.code == "tsts"
         "old"
       elsif @timeshifted and @set.code == "fut"
         "future"
-      else
+      elsif @release_date < m15_release_date
         # Were there any 8e+ old frame printings?
```

Comment:
A small number of Judge Promos, but they don't show up in mtgjson, so that's probably irrelevant.

Reply:
> A small number of Judge Promos, but they don't show up in mtgjson, so that's probably irrelevant.

So two things:

* this info belongs in mtgjson not here
* mtgjson really dropped the ball when MCI did. It just needs to accept that it has to be primary maintainer of promo information now.